### PR TITLE
Pull request for gnome-extra/gnome-commander (version bump)

### PR DESCRIPTION
**Pull request by turboscholz**
The pull request consists of three separate commits:
1. String fix in metadata.xml
2. Version bump to gnome-commander-1.6.0
3. Version bump to gnome-commander-1.6.1

The changes in the ebuild for 1.6.0 compared to 1.4.8 are explained in the associated commit message.

Merging this pull request would fix gentoo bug [#596952](https://bugs.gentoo.org/show_bug.cgi?id=596952).

### CODE REVIEW

File: gnome-extra/gnome-commander/gnome-commander-1.6.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,64 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
```
Comment (doughdemon) [inline gnome-extra/gnome-commander/gnome-commander-1.6.1.ebuild]: Please port to EAPI 6.

File: gnome-extra/gnome-commander/gnome-commander-1.6.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+	${RDEPEND}
+	dev-util/gtk-doc-am
+	>=dev-util/intltool-0.35.0
+	sys-devel/gettext
+	virtual/pkgconfig
+	test? ( >=dev-util/cppunit-1.13.2 )
+"
+
+src_configure() {
+	DOCS="AUTHORS BUGS ChangeLog NEWS README TODO"
```
Comment (doughdemon) [inline gnome-extra/gnome-commander/gnome-commander-1.6.1.ebuild]: Please put this into global scope. (See other ebuilds.)

**Comment by turboscholz** [comment]
Yeah, you are right, I removed version 1.6.0 from the pull request because it is not necessary. 

Update to EAPI=6 completed.

I checked the [documentation](https://devmanual.gentoo.org/eclass-reference/ebuild/) about DOCS and decided to remove the definition completely. The default (see `src_install` in the documentation) works well.

I furthermore removed version 1.4.2 - 1.4.7, as the current stable version is 1.4.8.

**Comment by doughdemon** [comment]
Changes look good to me.
Please squash your commits such that there are three commits: One for the version bump, one for the `metadata.xml` fix and the last removing the old ebuilds.

**Comment by turboscholz** [comment]
Done

**Comment by monsieurp** [comment]
ping?

**Pull request closed without merging**